edalize/templates/libero/libero-project.tcl.j2

```diff
@@ -0,0 +1,29 @@
+# This file is generated by Edalize.
+# Microsemi Tcl Script
+# Libero
+
+new_project -location {{op}}{{prj_root}}{{cl}} -name {{name}} -project_description {} -block_mode 0 -standalone_peripheral_initialization 0 -instantiate_in_smartdesign 1 -ondemand_build_dh 1 -hdl {{op}}{{tool_options.hdl}}{{cl}} -family {{op}}{{tool_options.family}}{{cl}} -die {{op}}{{tool_options.die}}{{cl}} -package {{op}}{{tool_options.package}}{{cl}} -speed {{op}}{{tool_options.speed}}{{cl}} -die_voltage {{op}}{{tool_options.dievoltage}}{{cl}} -part_range {{op}}{{tool_options.range}}{{cl}} -adv_options {IO_DEFT_STD:{{tool_options.defiostd}}{{cl}} -adv_options {RESTRICTPROBEPINS:1} -adv_options {RESTRICTSPIPINS:0} -adv_options {SYSTEM_CONTROLLER_SUSPEND_MODE:0} -adv_options {TEMPR:{{tool_options.range}}{{cl}} -adv_options {VCCI_1.2_VOLTR:EXT} -adv_options {VCCI_1.5_VOLTR:EXT} -adv_options {VCCI_1.8_VOLTR:EXT} -adv_options {VCCI_2.5_VOLTR:EXT} -adv_options {VCCI_3.3_VOLTR:EXT} -adv_options {VOLTR:{{tool_options.range}}{{cl}}
+
+set_device -family {{op}}{{tool_options.family}}{{cl}} -die {{op}}{{tool_options.die}}{{cl}} -package {{op}}{{tool_options.package}}{{cl}} -speed {{op}}{{tool_options.speed}}{{cl}} -die_voltage {{op}}{{tool_options.dievoltage}}{{cl}} -part_range {{op}}{{tool_options.range}}{{cl}} -adv_options {IO_DEFT_STD:{{tool_options.defiostd}}{{cl}} -adv_options {RESTRICTPROBEPINS:1} -adv_options {RESTRICTSPIPINS:0} -adv_options {SYSTEM_CONTROLLER_SUSPEND_MODE:0} -adv_options {TEMPR:{{tool_options.range}}{{cl}} -adv_options {VCCI_1.2_VOLTR:EXT} -adv_options {VCCI_1.5_VOLTR:EXT} -adv_options {VCCI_1.8_VOLTR:EXT} -adv_options {VCCI_2.5_VOLTR:EXT} -adv_options {VCCI_3.3_VOLTR:EXT} -adv_options {VOLTR:{{tool_options.range}}{{cl}}
```

Comment:
Remove this. Since we need to supply most of these parameters to new_project, I don't think we need a separate call to set_device

Reply:
Most scripts I saw create the project and also set the device. I'll test without the set_device command to see it it inherits from new_project.

Reply:
Give it a try. Maybe it is needed, but my experience tells me that in many cases these kind of scripts have just been copy-pasted around without any attempts at cleanup

Reply:
Just tested without the set_device and it loads fine. Already on last push.